### PR TITLE
update welcome message

### PR DESCRIPTION
**Pull request by jmikovic**

### CODE REVIEW

**Review by examon — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import { Application } from 'probot' // eslint-disable-line no-unused-vars
 
+const welcomeMessage: string = `Thank you for contributing to Leapp project!
+
+Please note that every PR needs to comply with the [Leapp Guidelines](https://github.com/oamg/leapp-guidelines), pass tests and linter check before it can be merged.
+If you want to re-run tests or request review, you can use following commands as a comment:
+- "leapp-ci build" - runs unit tests
+- "e2e test" - runs unit tests and end-to-end tests
```
Comment (pirat89) [inline src/index.ts]: Just regarding the recent discussion, people outside of oamg group will not be able to run those tests. From that point, shouldn't be here information like (oamg group only) or drop it completely?

File: src/index.ts [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+- "please review" - notifies leapp developers for review request`
```
Comment (Rezney) [inline src/index.ts]: "Notify of" rather than "notify for"...

File: src/index.ts [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import { Application } from 'probot' // eslint-disable-line no-unused-vars
 
+const welcomeMessage: string = `Thank you for contributing to Leapp project!
+
+Please note that every PR needs to comply with the [Leapp Guidelines](https://github.com/oamg/leapp-guidelines), pass tests and linter check before it can be merged.
+If you want to re-run tests or request review, you can use following commands as a comment:
+- "leapp-ci build" - runs unit tests
+- "please review" - notifies leapp developers of review request`
```
Comment (vojtechsokol) [inline src/index.ts]: I would also add the 'e2e tests' comment. Although the comment will trigger only for members of oamg team, it's good to have everything in one place (and even oamg members keep asking what the comment is...).

Comment (Rezney) [reply src/index.ts]: Wondering how did 'e2e tests' comment drop out. I think it was there initially...

Comment (pirat89) [reply src/index.ts]: my comments above ;-) we discussed that with Pepa in person as well and decided that drop seems better. But I do not care so much whether there will be the comment or nothing. From my point, we should know this thing everyone and we do not need to spam people. What if we add additional extra functions for us in future? But I am ok with both solutions.

Comment (jmikovic) [reply src/index.ts]: I knew I should've put it to cabal :D 
I didn't think this message was necessary for oamg members at all but if people are still struggling with the command then perhaps it makes sense to add e2e. We should do a quick poll or something...

**Review by fernflower — APPROVED** [review]

**Pull request merged**